miden-lib/asm/miden/kernels/tx/note.masm

```diff
@@ -120,3 +130,157 @@ export.prepare_note
     exec.memory::get_consumed_note_script_root
     # => [NOTE_SCRIPT_ROOT]
 end
+
```

Comment:
Did you change the procedures or just moved them? Don't know how to quickly check similarity

Reply:
Just moved them.